src/main/java/bio/terra/service/resourcemanagement/azure/AzureAuthService.java

```diff
@@ -121,9 +121,9 @@ public TableServiceClient getTableServiceClient(
 
   public TableServiceClient getTableServiceClient(AzureStorageAuthInfo storageAuthInfo) {
```

Comment:
Should we cover these lines with a unit test?

Reply:
Ticket here: https://broadworkbench.atlassian.net/browse/DCJ-446